### ISSUE TITLE
Why factory for simple object creation?

### ISSUE THREAD

**Issue by mniepieklo** [CertainGitHubUser/bc-test-assignment#4]
In order to create entity you're using factory, why? Wouldn't factory method in entity be more suitable?

**Comment by CertainGitHubUser** [comment]
@mniepieklo Agree. In this case, it's not needed. But usually, entities can be created from different arguments (smth like fromLegacyEntity, fromXDto, etc). So, the purpose was to show that I'm familiar with this pattern:)
